pkg/systemd/overview.jsx

```diff
@@ -0,0 +1,159 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import 'polyfills.js';
+import cockpit from "cockpit";
+import $ from "jquery";
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {
+    Page, PageSection, PageSectionVariants,
+    Gallery, Button,
+    Dropdown, DropdownItem, KebabToggle,
+} from '@patternfly/react-core';
+
+import { shutdown, shutdown_modal_setup } from "./shutdown.js";
+
+import { SystemInfomationCard } from './overview-cards/systemInformationCard.jsx';
+import { ConfigurationCard } from './overview-cards/configurationCard.jsx';
+import { HealthCard } from './overview-cards/healthCard.jsx';
+import { MotdCard } from './overview-cards/motdCard.jsx';
+import { UsageCard } from './overview-cards/usageCard.jsx';
+import { ServerTime } from './overview-cards/serverTime.js';
+
+import "form-layout.less";
+
+const _ = cockpit.gettext;
+var permission = cockpit.permission({ admin: true });
+$(permission).on("changed", update_shutdown_privileged);
```

Comment:
This new file only uses jQuery here and 3 lines down, both of which are trivial to replace with standard JS API.

Reply:
I fixed this but the 'update_shutdown_privileged'  still uses jquery and I am not sure how to replace it.

Reply:
should just be `document.querySelector(".shutdown-privileged").update_privileged(...)`; but if that doesn't work, nevermind -- let's clean it up later if it's nontrivial.